.github/workflows/ci.yml

```diff
@@ -53,7 +56,9 @@ jobs:
         go-version: '1.15'
     - uses: actions/checkout@v2
     - name: Run golangci-lint
-      run: make lint
+      run: |
+        curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin ${{ env.GOLANGCI_LINT_VERSION }}
```

Comment:
is there a benefit to installing CLI here instead of `Makefile`.

Reply:
Yes. This is specific for the CI. We would never want a `make lint` run to install something.

Reply:
> Yes. This is specific for the CI.

This is what I was trying to understand, thank you. 

> We would never want a `make lint` run to install something.

Agreed; I wasn't suggesting we do.